src/main/java/de/zalando/aruha/nakadi/controller/EventStreamController.java

```diff
@@ -71,7 +75,13 @@ public StreamingResponseBody streamEvents(
             final NativeWebRequest request, final HttpServletResponse response) throws IOException {
 
         return outputStream -> {
+
+            Counter consumerCounter = null;
+
             try {
+                consumerCounter = metricRegistry.counter(metricNameFor(eventTypeName, CONSUMERS_COUNT_METRIC_NAME));
+                consumerCounter.inc();
```

Comment:
I don't care. If it fails because of checks it will anyway remove that client immediately again. At this point it's a "potential" additional consumer.